exopy_hqc_legacy/manifest.enaml

```diff
@@ -388,7 +389,7 @@ enamldef HqcLegacyManifest(PluginManifest):
                     view = 'views.rf_views:RFFrequencyView'
                     instruments = ['exopy_hqc_legacy.Legacy.AgilentPSG',
                                    'exopy_hqc_legacy.Legacy.AnritsuMG3694',
-                                   'exopy_hqc_legacy.Legacy.LabBrickLMS103',
+                                   'exopy_qcircuits.Qcircuits.LabBrickLMS103',
```

Comment:
THIS has nothing to do HERE, it belongs inside QCircuits manifest.

Reply:
Agreed